manifests/dbfile.pp

```diff
@@ -33,12 +33,14 @@
 ) {
 
   file { "${postfixdir}/${title}":
-    owner   => $owner,
-    group   => $group,
-    mode    => $mode,
-    content => $content,
-    source  => $source,
-    ensure  => $ensure,
+    owner     => $owner,
+    group     => $group,
+    mode      => $mode,
+    content   => $content,
+    source    => $source,
+    ensure    => $ensure,
+    show_diff => false,
+    backup    => false,
```

Comment:
`show_diff => false` for postfix upstream to protect sasl password

This is to avoid displaying the content of `/etc/postfix/sasl_passwd`
on puppet client logs

`backup => false` is also needed to prevent sensitive files from
being saved on the clientbucket (usually `/var/lib/puppet/clientbucket`)
This is useful in a situation where you have an encrypted `/etc` but
`/var` is not encrypted